packages/carbon-web-components/src/components/tag/tag-story.ts

```diff
@@ -13,33 +13,69 @@ import { boolean, select } from '@storybook/addon-knobs';
 import textNullable from '../../../.storybook/knob-text-nullable';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { TAG_SIZE, TAG_TYPE } from './tag';
-import './filter-tag';
+import './index';
 import storyDocs from './tag-story.mdx';
 import { prefix } from '../../globals/settings';
 
-const noop = () => {};
-
 const sizes = {
-  'Regular size': null,
+  [`Medium size (${TAG_SIZE.MEDIUM})`]: TAG_SIZE.MEDIUM,
   [`Small size (${TAG_SIZE.SMALL})`]: TAG_SIZE.SMALL,
 };
 
-export const Default = (args) => {
-  const { size, type, title, disabled } = args?.[`${prefix}-tag`] ?? {};
+const types = [
+  'red',
+  'magenta',
+  'purple',
+  'blue',
+  'cyan',
+  'teal',
+  'green',
+  'gray',
+  'cool-gray',
+  'warm-gray',
+  'high-contrast',
+  'outline',
+];
+
+export const Default = () => {
+  return html`
+    ${types.map(
+      (e) => html`<cds-tag size="sm" type="${e}">Tag content</cds-tag>`
+    )}
+  `;
+};
+
+export const Playground = (args) => {
+  const { open, filter, size, type, title, disabled } =
+    args?.[`${prefix}-tag`] ?? {};
+
   return html`
     <cds-tag
+      ?filter="${filter}"
+      ?open="${open}"
       size="${ifDefined(size)}"
       type="${ifDefined(type)}"
       title="${ifDefined(title)}"
       ?disabled="${disabled}">
       This is a tag
+      <svg
```

Comment:
renderIcon looks good! but it seems like with the react code the filter icon should only render the close icon: https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Tag/Tag.js#L83. Which looks like what you have in the code, so maybe just a storybook update?

Reply:
Ohh yeah this was a leftover when i was testing the custom icon haha, thanks for noticing!